src/linagora.esn.calendar/app/event-view/external-user/event-view-external-user.controller.js

```diff
@@ -35,9 +35,10 @@ const _ = require('lodash');
       });
 
       $http({ method: 'GET', url: self.linksMapping[partstat] }).then(function() {
-        var participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
+        const participation = partstat.charAt(0).toUpperCase() + partstat.slice(1).toLowerCase();
+        const translatedParticipation = esnI18nService.translate(participation);
 
-        notificationFactory.weakInfo('Participation', esnI18nService.translate('Participation updated: %s', { participation }));
+        notificationFactory.weakInfo('Participation', esnI18nService.translate('Participation updated to:').toString().concat(translatedParticipation));
```

Comment:
`notificationFactory.weakInfo` service has translated the input string by itself. Better if having properly approach to avoid 2 times translating

Reply:
```  notificationFactory.weakInfo('Participation', esnI18nService.translate('Participation updated: %s', { participation })); ```
With this solution we can not translate participation properly because it is an enum type , so i suggest to translate it separately